src/Microsoft.DotNet.XHarness.TestRunners.Xunit/WasmThreadedTestRunner.cs

```diff
@@ -1,18 +1,29 @@
-﻿using System;
-using System.Collections.Generic;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Microsoft.DotNet.XHarness.Common;
 using Microsoft.DotNet.XHarness.TestRunners.Common;
+using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.DotNet.XHarness.TestRunners.Xunit;
 
 internal class WasmThreadedTestRunner : XUnitTestRunner
 {
     public WasmThreadedTestRunner(LogWriter logger) : base(logger)
     {
+        TestStagePrefix = string.Empty;
+    }
+
+    protected override string ResultsFileName { get => string.Empty; set => throw new InvalidOperationException("This runner outputs its results to stdout."); }
+
+    protected override void HandleTestFailed(ITestFailed msg)

```

Comment:
why is just this method overridden?

Reply:
The original implementation does a pretty extensive dump. We are logging less details in single threaded

```
info: [FAIL] System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success
info: [FAIL] System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success
info:    Test name: System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success(server: ws://127.0.0.1:60267/WebSocket/EchoWebSocket.ashx)
info:    Test case: System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success
info:    Assembly:  [System.Net.WebSockets.Client.Tests, Version=9.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51]
info:    Exception messages: Assert.True() Failure
info: Expected: True
info: Actual:   False
info:    Exception stack traces:    at System.Net.WebSockets.Client.Tests.SendReceiveTest.SendReceive_Concurrent_Success(Uri server) in C:\Development\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\tests\SendReceiveTest.cs:line 418
info: --- End of stack trace from previous location ---
info:    Execution time: 0.1452448
info:  **** Output start ****
info: GetConnectedWebSocket: ConnectAsync starting.
info: GetConnectedWebSocket: ConnectAsync done.
info:
info:  **** Output end ****
info:
info:    Test name: System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success(server: wss://127.0.0.1:60268/WebSocket/EchoWebSocket.ashx)
info:    Test case: System.Net.WebSockets.Client.Tests.ArraySegmentSendReceiveTest.SendReceive_Concurrent_Success
info:    Exception messages: Assert.True() Failure
info: Expected: True
info: Actual:   False
info:    Exception stack traces:    at System.Net.WebSockets.Client.Tests.SendReceiveTest.SendReceive_Concurrent_Success(Uri server) in C:\Development\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\tests\SendReceiveTest.cs:line 418
info: --- End of stack trace from previous location ---
info:    Execution time: 0.0540651
info:  **** Output start ****
info: GetConnectedWebSocket: ConnectAsync starting.
info: GetConnectedWebSocket: ConnectAsync done.
info:
info:  **** Output end ****
```

Reply:
I'd be inclined to fix this in the base class, we could log the additional output only in the Debug log case